setup.py

```diff
@@ -70,6 +70,7 @@
 # Check that this list is uptodate against the result of the command:
 # python bin/generate_module_list.py
 modules = [
+    'sympy.algebra',
```

Comment:
I think that the plural form `algebras` would be more suitable.

Reply:
I have been thinking for a while, both could be OK. I prefer to avoid plurals to make things simpler.

Reply:
But maybe in this case it's more appropriate to use the plural, as there will be probably many algebras.

Reply:
That is what I had in mind, too. It should also tell that not *all* algebraic matters are going to be included.

Reply:
We could also have `Complex` built on the same model: two component expressions with the familiar multiplication rules. That should not be confused with `ComplexFloat` that could exist in `core.numbers` and have `mpc` as representation.